kerberos_notes.md

```diff
@@ -0,0 +1,120 @@
+### Required environment for Kerberos authentication ###
```

Comment:
all the content from this file should be added to DEVELOPER_GUIDE.md. Also, please run it through grammar and spell-check.

Reply:
+1